src/server/game/Globals/ObjectMgr.h

```diff
@@ -657,7 +657,7 @@ typedef std::pair<ExclusiveQuestGroups::const_iterator, ExclusiveQuestGroups::co
 
 struct PetLevelInfo
 {
-    PetLevelInfo() : health(0), mana(0), armor(0)
+    PetLevelInfo() : health(0), mana(0), armor(0), min_dmg(0), max_dmg(0)
```

Comment:
minDamage or minimumDamage

Reply:
updated it in the parent branch #19409

Reply:
Most likely `minDamage` because 'minimumDamage' did not yield any results in a search through my local 3.3.5 files (https://github.com/TrinityCore/TrinityCore/commit/b507acf1c04ff6e9d0a075f9663e9549d5b238b4) or in the master branch (https://github.com/TrinityCore/TrinityCore/search?q=minimumDamage&utf8=✓) while `minDamage` gives search results both locally and in master (https://github.com/TrinityCore/TrinityCore/search?q=minDamage&utf8=✓).

Reply:
thanks anyway!